### PR TITLE
Add BFT space

### PR DESCRIPTION
**Pull request by BFTFinance**

### CODE REVIEW

File: spaces/bft/index.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "key": "bft",
+  "symbol": "BFT",
+  "name": "Berkshire.Finance",
```
Comment (e-asia) [inline spaces/bft/index.json:4]: Can you choose a shorter name? This will break the interface on a small screen otherwise.

Comment (BFTFinance) [reply spaces/bft/index.json]: Yes, sorry about that just made the change. -Thank you

File: spaces/bft/index.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "key": "bft",
+  "symbol": "BFT",
+  "name": "BFT Finance",
+  "defaultView": "all",
+  "address": "0x1ec128C3D3F256Cad161cf7A3F148f47F03f1D36",
+  "token": "0x1ec128C3D3F256Cad161cf7A3F148f47F03f1D36",
```
Comment (e-asia) [inline spaces/bft/index.json:7]: Something is wrong with the token, the call to query decimals() is failing. Is this a standard ERC20 token?

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/bft/index.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "BFT Finance",
+  "defaultView": "all",
+  "address": "0x1ec128C3D3F256Cad161cf7A3F148f47F03f1D36",
+  "token": "0x1ec128C3D3F256Cad161cf7A3F148f47F03f1D36",
+  "core": [
+    "0x7eB2a7f798b01E9BCc1C158140b8c8E036205233",
+  ],
+  "min": 10000,
+  "invalid": []
+}
```
Comment (e-asia) [inline spaces/bft/index.json:13]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/bft/index.json (hunk 3 above)
Comment (BFTFinance) [reply spaces/bft/index.json:15]: Thank you, have made the updates based on the format.

File: spaces/bft/index.json (hunk 2 above)
Comment (e-asia) [reply spaces/bft/index.json:9]: @BFTFinance there is always a problem with the token.

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by e-asia** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**